<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>818</width>
    <height>468</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>818</width>
    <height>468</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QWidget" name="widget_status" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>439</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>791</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_pushbutton" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>52</y>
     <width>795</width>
     <height>368</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>795</width>
     <height>368</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>795</width>
     <height>368</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="SlidePushButton" name="pushButton_serial">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>161</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="SlidePushButton" name="pushButton_network">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>161</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="SlidePushButton" name="pushButton_plot">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>189</width>
        <height>161</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="SlidePushButton" name="pushButton_setting">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>161</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="SlidePushButton" name="pushButton_protocal">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>161</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="SlidePushButton" name="pushButton_state">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>161</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="SlidePushButton" name="pushButton_control">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>189</width>
        <height>161</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="SlidePushButton" name="pushButton_about">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>161</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>9</y>
     <width>665</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_arrow">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_back">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../res/res.qrc">
          <normaloff>:/pushbutton_arrow/main_image/pushbutton_arrow/back.png</normaloff>:/pushbutton_arrow/main_image/pushbutton_arrow/back.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_foward">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../res/res.qrc">
          <normaloff>:/pushbutton_arrow/main_image/pushbutton_arrow/foward.png</normaloff>:/pushbutton_arrow/main_image/pushbutton_arrow/foward.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>68</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_network">
      <item>
       <widget class="QLabel" name="label_network_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Network:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SwitchButton" name="widget_toggle_network" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_serial">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Serial</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SwitchButton" name="widget_toggleserial" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_network">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../res/res.qrc">:/network_state/main_image/network_state/no_connect.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_speed">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>speed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SlidePushButton</class>
   <extends>QPushButton</extends>
   <header>util/sliding/slide_pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>util/widget/switch_button.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
